cpu/stm32/vectors/vectors_f3.c

```diff
@@ -244,9 +244,9 @@ ISR_VECTOR(1) const isr_t vector_cpu[CPU_IRQ_NUMOF] = {
     (0UL),                  /* [79] Reserved */
     (0UL),                  /* [80] Reserved */
     isr_fpu                 /* [81] Floating point Interrupt */
-#elif (CPU_LINE_STM32F334x8)
+#elif defined(CPU_LINE_STM32F334x8)
```

Comment:
The changes in this file should be provided in a separate commit (same for the board config and the change in spi.c by the way).
Note that this will be obsolete once #14595 is merged (if it is merged one day :) ! )